runners/direct-java/src/main/java/org/apache/beam/runners/direct/StatefulParDoEvaluatorFactory.java

```diff
@@ -278,15 +300,41 @@ public void processElement(WindowedValue<KeyedWorkItem<K, KV<K, InputT>>> gbkRes
 
     @Override
     public TransformResult<KeyedWorkItem<K, KV<K, InputT>>> finishBundle() throws Exception {
+
       TransformResult<KV<K, InputT>> delegateResult = delegateEvaluator.finishBundle();
+      Boolean isTimerDeclared = false;
```

Comment:
No need to use boxed type here - use boolean instead

Reply:
done